src/components/ErrorNotification/ErrorNotification.tsx

```diff
@@ -0,0 +1,53 @@
+/* eslint-disable jsx-a11y/control-has-associated-label */
+import React, { useEffect } from 'react';
+import { Errors } from '../../types/Errors';
+
+type Props = {
+  isError: boolean
+  onSetIsError: React.Dispatch<React.SetStateAction<boolean>>
+  typeError: string
+};
+
+export const ErrorNotification: React.FC<Props> = ({
+  isError,
+  onSetIsError,
+  typeError,
+}) => {
+  useEffect(() => {
+    onSetIsError(isError);
+    if (isError) {
+      setTimeout(() => {
+        onSetIsError(true);
+      }, 3000);
+    }
+  }, [isError]);
```

Comment:
Your errror don't disappear in 3 seconds. Maybe you need to clearTimeout  if it was set.
https://prnt.sc/49poofCDtBVt